images/win/toolset-2016.json

```diff
@@ -74,5 +74,41 @@
                 "1.72.0"
             ]
         }
+    ],
+    "azureModules": [

```

Comment:
Should not we define the default version?

Reply:
Do you mean `azure 2.1.0` and `azurerm 2.1.0`? I'm not sure, if we have default version for az.

Reply:
Yep, only two default modules in the path azure 2.1.0 and azurerm 2.1.0

Reply:
Done, thanks!